ChatSecure/Classes/Controllers/XMPP/OTRXMPPRoomManager.m

```diff
@@ -133,6 +133,27 @@ - (void)inviteUser:(XMPPJID *)user toRoom:(XMPPJID *)roomJID withMessage:(NSStri
     [room inviteUser:user withMessage:message];
 }
 
+- (void)inviteBuddies:(NSArray<NSString *> *)buddyUniqueIds toRoom:(XMPPRoom *)room {
+    if (!buddyUniqueIds.count) {
+        return;
+    }
+    NSMutableArray<OTRXMPPBuddy*> *buddies = [NSMutableArray arrayWithCapacity:buddyUniqueIds.count];
+    [self.inviteDictionary removeObjectForKey:room.roomJID.bare];
```

Comment:
What is the `inviteDictionary` doing here exactly? There is a risk of mutating it from multiple threads. I think it should be moved back to where it was extracted below.